lexiblaster/index.html

```
@@ -32,79 +32,87 @@
+        <canvas id="gameCanvas" width="800" height="720"></canvas>
+      </div>
+      <p class="game-links"><a href="#leaderboard">ランキングを見る</a> / <a href="/">Irori's Toybox トップ</a></p>
+      <p class="custom-mode-cta">
+        <button id="customModeBtn" type="button" class="custom-mode-btn">🛠 カスタムモード（自分の単語で練習）</button>
+      </p>
+      <!-- 共有ボタン（ゲーム終了時に表示） -->
+      <button id="shareXBtn" class="share-x-btn" hidden>🡕 Xで共有</button>
+      <button id="shareLineBtn" class="share-line-btn" hidden>🟩 LINEで共有</button>
+    </section>
+
+  <div id="wrap">
```

HTMLHint (1.9.2)

[error] The id value [ wrap ] must be unique.

(id-unique)